src/builtin.c

```diff
@@ -657,6 +657,48 @@ static jv f_format(jq_state *jq, jv input, jv fmt) {
     }
     jv_free(input);
     return line;
+  } else if (!strcmp(fmt_s, "urid")) {
+    jv_free(fmt);
+    input = f_tostring(jq, input);
+
+    jv line = jv_string("");
+    const char *errmsg =  "is not a valid uri encoding";
+    char *s = (char *)jv_string_value(input);
```

Comment:
I don't see any reason why you need to cast away the const instead of just declaring the variable `const char *`

Reply:
lol good point idk why i had this